components/ai_rewriter/common/BUILD.gn

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+component("common") {
+  output_name = "ai_rewriter_common"
+  defines = [ "IS_AI_REWRITER_COMMON_IMPL" ]
+
+  sources = [
+    "features.cc",
+    "features.h",
+  ]
+
+  deps = [ "//brave/components/ai_chat/core/common" ]
+
+  public_deps = [ "//brave/components/ai_rewriter/common/mojom" ]
```

Comment:
Super nit: public deps should be before deps

https://chromium.googlesource.com/chromium/src/tools/gn/+/48062805e19b4697c5fbd926dc649c78b6aaa138/docs/style_guide.md#ordering-within-a-target